.github/workflows/functional-test.yaml

```diff
@@ -383,7 +383,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-go-
       - name: Download rad CLI
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
```

Comment:
This is the reason why the functional test failed. I used actions/download-artifact@v3, but uploading artifact uses v4 version.